aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -98,24 +75,32 @@ func TestAccAWSElasticacheReplicationGroup_basic(t *testing.T) {
 			{
 				Config: testAccAWSElasticacheReplicationGroupConfig(rName),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckAWSElasticacheReplicationGroupExists("aws_elasticache_replication_group.bar", &rg),
+					testAccCheckAWSElasticacheReplicationGroupExists("aws_elasticache_replication_group.test", &rg),
```

Comment:
Nit: I do not want to hold up this PR over it, but these should be using the new `resourceName` variable